SwiftUIDev/SwiftUIDev/Spotify/Screens/LoginView.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  LoginView.swift
+//  SwiftUIDev
+//
+//  Created by LeeJiSoo on 2022/08/07.
+//
+
+import SwiftUI
+
+struct LoginView: View {
+    let buttonHorizontalPadding = 32.0
+    let mainLogoSize = 44.0
+    let signupButtonInterval = 10.0
+    let loginButtonPadding = 28.0
+    let titleInterval = 28.0
+    let titleAndButtonInterval = 40.0
```

Comment:
```suggestion
private enum Size {
    static let buttonHorizontalPadding = 32.0
    static let mainLogoSize = 44.0
    static let signupButtonInterval = 10.0
    static let loginButtonPadding = 28.0
    static let titleInterval = 28.0
    static let titleAndButtonInterval = 40.0
}
```
공통으로 관리되지 않는 nameSpace라면 위와같은 방법은 어떨까요 ??

nameSpace에 대해 관련 글 첨부합니다 한번 참고해보세용 ! ^______^

https://zeddios.tistory.com/353

https://sujinnaljin.medium.com/namespace-%EC%A7%80%EC%A0%95%ED%95%98%EA%B8%B0-struct-vs-enum-ce86d5717d95